src/ServiceManagement/Compute/Commands.ServiceManagement.Preview/Automation/RunCodeGeneration.ps1

```diff
@@ -1317,6 +1374,31 @@ function Write-OperationCmdletFile
             $paramTypeNormalizedName = Get-NormalizedTypeName -inputName $paramTypeFullName;
             $param_constructor_code = Get-ConstructorCodeByNormalizedTypeName -inputName $paramTypeNormalizedName;
 
+            $is_special_type = Is-SpecialType -inputName $paramTypeFullName;
```

Comment:
The term ```is_special_type``` is too generic. Consider using more specific term like "is model class type" etc.